qontract_utils/tests/test_quay_api.py

```
@@ -0,0 +1,254 @@
+        json={"visibility": "private"},
+    )
+
+
+# ---------------------------------------------------------------------------
+# QuayApiCallContext
+# ---------------------------------------------------------------------------
+
+
+def test_call_context_is_frozen() -> None:
+    ctx = QuayApiCallContext(method="repository.list", verb="GET", org="my-org")
+    with pytest.raises(Exception):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)